templates/miq-template-ext-db.yaml

```diff
@@ -55,7 +55,7 @@ objects:
         ProxyPass /api http://web-service:3000/api
         ProxyPassReverse /api http://web-service:3000/api
 
-        # Ensures httpd stdout/stderr are seen by docker logs.
```

Comment:
In this context `docker logs` is a command.

Reply:
Didn't realize that's a command... I'll put a quote around it then.